### ISSUE TITLE
My game crashed

### ISSUE THREAD

**Issue by neilmunday** [OpenRA/OpenRA#18210]
## System Information

- Operating System:  Windows 10 (64-bit)
- OpenRA Version: release-20200202
- OpenRA Mod: Red Alert
- Source: Official download package

## Exception log
```
OpenRA engine version release-20200202
Red Alert mod version release-20200202
on map 12304e68fd91e8b602ae1967ad9f4c9a51733372 (Naval Wars by psaez).
Date: 2020-05-30 15:58:20Z
Operating System: Windows (Microsoft Windows NT 6.2.9200.0)
Runtime Version: .NET CLR 4.0.30319.42000
Exception of type `System.InvalidOperationException`: Attempting to query the position of an invalid Target
   at OpenRA.Traits.Target.get_CenterPosition()
   at OpenRA.Mods.Common.Activities.FlyAttackRun.OnFirstRun(Actor self)
   at OpenRA.Activities.Activity.TickOuter(Actor self)
   at OpenRA.Traits.ActivityUtils.RunActivity(Actor self, Activity act)
   at OpenRA.Activities.Activity.TickChild(Actor self)
   at OpenRA.Activities.Activity.TickOuter(Actor self)
   at OpenRA.Traits.ActivityUtils.RunActivity(Actor self, Activity act)
   at OpenRA.Actor.Tick()
   at OpenRA.World.Tick()
   at OpenRA.Game.InnerLogicTick(OrderManager orderManager)
   at OpenRA.Game.LogicTick()
   at OpenRA.Game.Loop()
   at OpenRA.Game.Run()
   at OpenRA.Game.InitializeAndRun(String[] args)
   at OpenRA.WindowsLauncher.RunGame(String[] args)
```

## Additional information
It was 2 human, 4 AI session. Both humans on same team against 4 AI all on the same team.

**Comment by pchote** [comment]
Thanks for the report. This crash was fixed in the 20200503 release, which can be downloaded from the website.

**Comment by neilmunday** [comment]
Thanks for the info - my bad for not seeing there was a new release available before we started playing.
